requirements.txt

```diff
@@ -1,21 +1,21 @@
 boto3<=1.28.33     # <= is used for python3.6 compatibility on centos-7
 botocore<=1.31.33  # <= is used for python3.6 compatibility on centos-7
-cachetools==4.2.4
-certifi==2024.7.4
-charset-normalizer==2.0.12
-google-auth==2.13.0
-idna==3.7
-kubernetes==25.3.0
-oauthlib==3.2.2
-pyasn1==0.4.8
-pyasn1-modules==0.2.8
-python-dateutil==2.8.2
-PyYAML==6.0.1
-requests==2.32.0
-requests-oauthlib==1.3.1
-rsa==4.9
-six==1.16.0
-urllib3==1.26.19
-websocket-client==1.3.1
-ply==3.11
-tornado==6.4.2
+cachetools>=4.2
+certifi>=2024.7
+charset-normalizer>=2.0
+google-auth>=2.13
+idna>=3.7
+kubernetes>=25.3
+oauthlib>=3.2
+pyasn1>=0.4
+pyasn1-modules>=0.2
+python-dateutil>=2.8
```

Comment:
We could try using the `~= X.Y` syntax, which stops at major version changes (major version bumps are likely to break APIs, minor and patch version updates should be enough for rebuilds).

Reply:
Thanks for the notice, I have changed this part